lumigator/python/mzai/backend/backend/api/routes/health.py

```diff
@@ -46,9 +51,24 @@ def get_job_metadata(job_id: UUID) -> JobSubmissionResponse:
             detail="Invalid JSON response"
         ) from e
 
+@router.get("/jobs/{job_id}/logs")
+def get_job_logs(job_id: UUID) -> JobLogsResponse:
+    resp = requests.get(ray_jobs() + f"{job_id}/logs")
```

Comment:
One thing I don't like is that urllib handles mainly str params instead of the specialized type for urls, afaict. But I guess that's better than joining the urls together, ok.